lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+const defaultLogo = require('../../assets/img/logo_nexus_generic.svg');
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+import './NxNexusPageHeader.scss';
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href} data-text={name}>{name}</a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxNexusPageHeader({ links, homeLink, productInfo, children, logoPath }: Props) {
+  const logoImg = logoPath || defaultLogo,
+      logoEl = <img src={logoImg} className="nx-product__logo-image" />,
+      linkEls = links && links.map(link => <HeaderLink key={link.name} { ...link } />);
+
+  return (
+    <header className="nx-page-header nx-page-header--nexus">
+      <div className="nx-page-header__inner">
+        <div className="nx-product">
+          <div className="nx-product__logo">
+            { homeLink ? <a className="nx-product__link" href={homeLink}>{logoEl}</a> : logoEl }
+          </div>
+          <div className="nx-product__summary">
+            <span className="nx-product__brand">nexus</span>
+            { productInfo && <span className="nx-product__name">{productInfo.name}</span> }
+          </div>
+          <div className="nx-product__details">
+            <span className="nx-product__byline">from sonatype</span>
+            { productInfo &&
+              <>
+                <span className="nx-product__meta">{productInfo.meta}</span>
+              </>
+            }
+            { productInfo &&
+              <>
+                <span className="nx-product__version">{productInfo.version}</span>
+              </>
```

Comment:
These don't really need to be separate spans.  If they were one, the `|` would be simpler to implement.

Reply:
> These don't really need to be separate spans. If they were one, the `|` would be simpler to implement.

Yeah that is better. I stayed with "meta" since the span is now containing two pieces of info about the product: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/a1c0dbdf21a345b3b2d56fb6485076c12155bd8a